src/platform/vita/vita_input.c

```diff
@@ -0,0 +1,428 @@
+#include "vita_input.h"
+#include "vita_touch.h"
+#include <math.h>
+
+#include "vita_keyboard.h"
+#define DISPLAY_WIDTH 960
```

Comment:
Defined several times, put it in a header file and prefix it with `VITA`.

Reply:
Fixed in 35f42ab00e2b28a49e966a752eb3016a334d53bb